frontend/src/component/Search/CheckboxFilter.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { translate } from 'react-i18next'
+import PropTypes from 'prop-types'
+import { Icon } from 'tracim_frontend_lib'
+
+export const CheckboxFilter = props => {
+  const { props, state } = this
+  return (
+    <>
+      <div className='searchFilterMenu__content__item__title'>
+        <button
+          className='transparentButton'
+          onClick={props.onClickOpenOrCloseFilter}
+        >
+          <Icon
+            icon={props.showFilter
+              ? 'fa-fw fas fa-caret-down'
+              : 'fa-fw fas fa-caret-right'}
+            title={props.showFilter
+              ? props.t('Hide {{filter}}', { filter: props.label })
+              : props.t('Show {{filter}}', { filter: props.label })}
+          />
+        </button>
+        {props.label}
+      </div>
+
+      {props.showFilter &&
+        props.filterList.map(item =>
```

Comment:
try to match the column position of the opening and closing bracket and parenthesis.
You can either put props.filterList at the same line or add an opening parenthesis after the &&

Reply:
fix